server/src/pages/leave-screenshots/view.js

```diff
@@ -29,7 +29,7 @@ class Body extends React.Component {
     }
     return (
       <reactruntime.BodyTemplate {...this.props}>
-        <div className="column-center full-height">
+        <div className="column-center full-height" role="main">
```

Comment:
Curious why you're using role=main in some cases, and the main element in others?

Reply:
In most of the pages we have main content inside single div and setting attribute role as main suffice. On homepage our main content is spread out in multiple tags and thast why needs a containing main tag